perun-rpc/src/main/perl/listOfHostNumCores

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Text::ASCIITable;
+use Perun::Agent;
+use Perun::Common qw(printMessage tableToPrint getSortingFunction);
+
+sub help {
+	return qq{
+	Lists hostnames and their number of cores.
+	---------------------------------------------------
+	Available options:
+	--facilityName | -F  facility name
+	--facilityId   | -f  facility Id
+	--orderById    | -i  order by ID
+	--orderByName  | -n  order by name
+	--batch        | -b  batch
+	--help         | -h  prints this help
+        
+        If no facility is entered, list for all facilities is displayed.
+	};
+}
+
+my ($facilityName, $facilityId, $all, $sortingFunction, $batch, @facilities, @hosts);
+GetOptions("help|h" => sub { print help; exit 0;} ,
+"facilityId|f=i" => \$facilityId,
+"facilityName|F=s" => \$facilityName,
+"orderByName|n" =>  sub {$sortingFunction = getSortingFunction("getHostname", 1)},
+"orderById|i" => sub { $sortingFunction = getSortingFunction("getId")},
+"batch|b" => \$batch) || die help;
+
+#options check
+unless(defined $facilityId or (defined $facilityName)) { $all=1;} else {$all=0;} 
+unless(defined $sortingFunction) { $sortingFunction = getSortingFunction("getHostname", 1); }
+
+my $sortingFunction2 = getSortingFunction("getName",1); 
+my $attrId=400;
```

Comment:
I would prefer getting attribute by its URN and not ID, ID is specific to instance of DB, name is not. You can get attribute ID from retrieved attribute definition.